poc/extensions/src/extension_fungibles.rs

```diff
@@ -0,0 +1,42 @@
+use crate::{DispatchError, Dispatchable};
+use crate::{ExtensionId, ExtensionIdTy};
+use core::marker::PhantomData;
+use parity_scale_codec::{Decode, Encode};
+
+pub trait ExtensionFungibles: ExtensionId {
```

Comment:
```suggestion
pub trait ExtensionFungibles {
```

it is not up to the implementor of `ExtensionFungibles` to determine the extension id